app/controllers/hours_controller.rb

```diff
@@ -21,6 +21,7 @@ def update
   def edit
     super
     resource
+    @last_hour_logged = current_user.hours.last
```

Comment:
Don't you think that this means that when a user selects a particular entry(hour) to edit, instead of prefilling the selected one, the last hour is selected, which might not be the same as what was selected?.